htdocs/projet/element.php

```diff
@@ -1164,7 +1164,7 @@
 			for ($i = 0; $i < $num; $i++) {
 				$tmp = explode('_', $elementarray[$i]);
 				$idofelement = $tmp[0];
-				$idofelementuser = $tmp[1];
+				$idofelementuser = $tmp[1] ?? null;
```

Comment:
If you include this code it means at next line, you may make a fetch with null parameters. If value is null, there is no reason to make a fetch
Can you describe a step to reproduce the warning ?

Reply:
`$elementarray` comes from `get_element_list()`, and is formatted according to line 865 of `htdocs/projet/class/project.class.php`, which makes the `'_'.$obj->fk_user` suffix optional:

```
    $elements[$i] = $obj->rowid.(empty($obj->fk_user) ? '' : '_'.$obj->fk_user);
```

We're working with invoices, orders, and tasks that are imported from another system by a script, and therefore don't have fk_user set.